### PR TITLE
Save feedback to postgres

### PR DESCRIPTION
**Pull request by coffee-cup**
- Setup prisma
- Save feedback responses to Postgres
- Only send negative resopnses to Discord
- Split feedback UI into separate component
- Fix z-index of model

### CODE REVIEW

File: src/pages/api/feedback.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  feedback?: string;
+}
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method == "POST") {
+    const body: FeedbackBody = JSON.parse(req.body);
+    const feedbackMessage =
+      body.feedback?.trim() === "" ? null : body.feedback?.trim();
+
+    const discordBody = {
```
Comment (FarazPatankar) [inline src/pages/api/feedback.ts:17]: Maybe create this only if we're sending the message to Discord?

**Review by FarazPatankar — APPROVED** [review]
LGTM

File: prisma/migrations/20210915164107_feedback/migration.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-- CreateEnum
+CREATE TYPE "Sentiment" AS ENUM ('POSITIVE', 'NEGATIVE');
+
+-- CreateTable
+CREATE TABLE "Feedback" (
+    "id" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```
Comment (ndneighbor) [inline prisma/migrations/20210915164107_feedback/migration.sql:8]: For my knowledge, why would there be an updated at if there is no editing on these comments

**Review by ndneighbor — APPROVED** [review]

File: prisma/migrations/20210915164107_feedback/migration.sql (hunk 2 above)
Comment (coffee-cup) [reply prisma/migrations/20210915164107_feedback/migration.sql:8]: It probably won't be updated, but we add createdAt and updatedAt to all our models as a convention

**Pull request merged**